specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/stable/2020-03-01/inputs.json

```diff
@@ -0,0 +1,1050 @@
+          "description": "The properties that are associated with the Avro serialization type. Required on PUT (CreateOrReplace) requests.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AvroSerializationProperties": {
+      "description": "The properties that are associated with the Avro serialization type.",
+      "properties": {}
+    },
+    "Encoding": {
+      "type": "string",
+      "description": "Specifies the encoding of the incoming data in the case of input and the encoding of outgoing data in the case of output.",
+      "enum": [
+        "UTF8"
+      ],
+      "x-ms-enum": {
+        "name": "Encoding",
+        "modelAsString": true
+      }
+    },
+    "JsonOutputSerializationFormat": {
+      "type": "string",
+      "description": "Specifies the format of the JSON the output will be written in. The currently supported values are 'lineSeparated' indicating the output will be formatted by having each JSON object separated by a new line and 'array' indicating the output will be formatted as an array of JSON objects.",
+      "enum": [
+        "LineSeparated",
+        "Array"
+      ],
+      "x-ms-enum": {
+        "name": "JsonOutputSerializationFormat",
+        "modelAsString": true
+      }
+    },
+    "ResourceTestStatus": {
+      "description": "Describes the status of the test operation along with error information, if applicable.",
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The status of the test operation."
+        },
+        "error": {
+          "readOnly": true,
+          "$ref": "#/definitions/ErrorResponse",
+          "description": "Describes the error that occurred."
+        }
+      }
+    },
+    "ErrorResponse": {
+      "description": "Describes the error that occurred.",
+      "properties": {
+        "code": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Error code associated with the error that occurred."
+        },
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Describes the error in detail."
+        }
+      }
+    },
+    "SubResource": {
+      "description": "The base sub-resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "Compression": {
+      "description": "Describes how input data is compressed",
+      "properties": {
+        "type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "Identity": {
```

Comment:
>Identity [](http://example.com/codeflow?start=5&length=8)

This definition appears to be unreferenced.

Reply:
Actually, I just found the reference over in streamingJobs.json. Might want to consider moving it over there, since it's referenced there and not in this file.